src/globals/scss/_theme-tokens.scss

```diff
@@ -200,6 +200,10 @@
   $data-table-heading-border-bottom: 1px solid $brand-01 !default !global;
   $data-table-row-height: 2rem !default !global;
 
+  // Date Picker
+  $in-range-background-color: $ibm-colors__blue--20 !default !global;
```

Comment:
Would we need to prefix these tokens (thank you for adding BTW!) with the component name, to avoid variable conflicts...?

Reply:
Ah right, good catch! The plan is to go over all these one-off variables with the designers and try to create new reusable color tokens, at a later date. For now, I'll update this to be prefixed with the component name 👍